GuruLoader/GuruLoader.cs

```diff
@@ -80,6 +80,25 @@ public class DisplayPortfolio {
     public IEnumerable<DisplayPosition> Positions { get; set; }
 }
 
+public class HyperPosition {
+    public string Name { get; set; }
+    public string ClassTitle { get; set; }
+    public string Cusip { get; set; }
+    public string PutCall { get; set; }
+    public double PercOfPortfolio { get; set; }
+    public int NumberGurusOwning { get; set; }
```

Comment:
It can't. Once you receive an hyperportfolio (i.e. from a json call) there is no way to calculate the total number of gurus involved by looking at the data in the hyperpositions. Not sure if I explained myself well enough. If not let's chat.